libtransmission/blocklist.h

```diff
@@ -9,19 +9,93 @@
 #error only libtransmission should #include this header.
 #endif
 
+#include <string>
+#include <vector>
+
+#include "file.h" // for tr_sys_file_t
+#include "tr-assert.h"
+#include "tr-macros.h"
+
 struct tr_address;
-struct tr_blocklistFile;
 
-tr_blocklistFile* tr_blocklistFileNew(char const* filename, bool isEnabled);
+struct BlocklistFile
+{
+public:
+    BlocklistFile(char const* filename, bool isEnabled)
+        : is_enabled_(isEnabled)
+        , filename_(filename)
+    {
+    }
+
+    ~BlocklistFile();
+
+    void close();
```

Comment:
It looks like this is only called by ~BlocklistFile and by BlockListfFile::load(), so it could be private

Reply:
Moving to private